packages/runtime/src/templates/edge/shims.js

```diff
@@ -48,6 +49,16 @@ const fetch /* type {typeof globalThis.fetch} */ = async (url, init) => {
   }
 }
 
+// Turbopack aliases "Buffer" to a module import, so we need to provide a shim for that
+if (typeof require === 'undefined') {
```

Comment:
This is the work in https://github.com/netlify/next-runtime/pull/2050. We can get your PR in tomorrow @orinokai and keep working in here for the upgrade.